### ISSUE TITLE
[Bug]: Platforms have no reason to be climbable and walkable, as well as no option to remove or add

### ISSUE THREAD

**Issue by FabianK3** [Aurorastation/Aurora.3#19708]
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

Platforms currently represent multiple issue in every aspect.

![platform](https://github.com/user-attachments/assets/56381c49-c6d8-4867-98f5-a1c6d9564992)

- The platforms are walkable which is very confusing because just based of their optics, you don't really trust them.
- The platforms have to be climbed like a railing or desk to get over them, which is a really weird and dangerous behavior considering they are always near an open space
- You cannot remove them
- You cannot add new platforms

In a nutshell: Their behavior is not practical in any way, especially for engineering.

EDIT: Additionally you can't pull crates or canisters over them.

### How to reproduce

Nearly every open space around medical or the central ring feature platforms. Every platform behaves the same.

**Comment by FabianK3** [comment]
I've come to realize that platforms being climbable/non-passable is actually a good design like for stages or elevated areas.

I would suggest finding another solution for edges like those seen in the screenshot nonetheless as their behavior doesn't fit their location like e.g. a elevated stage does.
